Makefile

```diff
@@ -161,9 +161,12 @@ fmt: ## Check formatting of go sources.
 yamllint:
 	yamllint -c .yamllint deploy/examples/ --no-warnings
 
+.PHONY: lint
+lint: yamllint pylint shellcheck vet ## Run various linters
```

Comment:
Hot take: vet *is* a linter, *but* it's better to run it with unit tests because `vet` and `test` both require go code to be compiled. Running them together will gain some speedup in certain cases, and Rook takes quite a while to compile.

Reply:
@BlaineEXE  right, I agree, and best I can tell, vet is already an implicit dependency of `make test`. 

But for me, linting isessentially checking  for syntactic correctness. In that view,  `go vet` is performing some  linting. Therefore, I wanted to include it in the `make lint`umbrella. The same would apply to `golangci-lint` which includes some go vetting functionality and which I yet want to extract into a make target.